packages/komencikit/src/kit.ts

```diff
@@ -2,11 +2,11 @@ import { Address, normalizeAddressWith0x, serializeSignature, sleep } from '@cel
 import { Err, Ok, Result } from '@celo/base/lib/result'
 import { CeloTransactionObject, CeloTxReceipt } from '@celo/connect'
 import { ContractKit } from '@celo/contractkit'
-import { ContractEventLog } from '@celo/contractkit/lib/generated/types'
```

Comment:
@gastonponti maybe you have more insight on this, but I think e might have made some changes to the `tsconfig` and the generated type files (`.d.ts`) aren't copied to `lib` when the package is built, so I had to change this import to the `src` folder. 

I don't think it's a problem in this case but I wanted to make sure that we know this is happening and it's intended behaviour.

Reply:
Mmmm, are you using a fixed version of the contractkit?
This was part of a problem that I'm pretty sure that was fixed. We forgot to add those types in the `@celo/typechain-target-web3-v1-celo` package (that the version `-beta3` of that typechain should be the one used in protocol right now in master), and was the one used to release the `contractkit-beta4`

If you are using a fixed version, use the -beta4 one (soon the 1.0.0, but still on testing)